### ISSUE TITLE
The UI sometimes does not display validation errors

### ISSUE THREAD

**Issue by lkuchenb** [ghga-de/datameta#281]
Occasionally (as in half of the time 🙈 ) the submit UI fails to visualize the verification errors that occurred in the sample sheet

Instead of the error table container
<img width="926" alt="Screen Shot 2021-04-14 at 16 19 34" src="https://user-images.githubusercontent.com/5948880/114725921-356f8900-9d3d-11eb-8a4a-c6eb61170ae7.png">
a generic error is shown
<img width="927" alt="Screen Shot 2021-04-14 at 16 19 02" src="https://user-images.githubusercontent.com/5948880/114725840-225cb900-9d3d-11eb-9f76-20deee4e22be.png">
and the JS console shows the following exception
```
Error: TOOLTIP: Option "content" provided type "null" but expected type "(string|element|function)".
    at index.js:119
    at Array.forEach (<anonymous>)
    at p (index.js:111)
    at n.r._getConfig (tooltip.js:712)
    at n.o [as constructor] (tooltip.js:144)
    at new n (popover.js:67)
    at submit.js:161
    at Array.map (<anonymous>)
    at Object.DataMeta.reloadPopovers (submit.js:160)
    at Object.DataMeta.submit.visualizeErrors (submit.js:396)
```
So it appears to be related to the generation of the tooltips (that pop up when one clicks a yellow exclamation mark)

https://getbootstrap.com/docs/5.0/components/tooltips/

**Comment by MoritzHahn1337** [comment]
Only happens when there are correct as well as incorrect lines in the excel sheet

**Comment by lkuchenb** [comment]
I can reliably trigger this with the `lastest_error.xlsx` file I shared with you 🤔

**Comment by MoritzHahn1337** [comment]
Thats even more interesting.
I tracked it down to be indeed some kind of race condition but have not found the solution. When debugging step by step, it never happens :-D

**Comment by lkuchenb** [comment]
Fixed in #284
